### PR TITLE
Re-enables space runes on Station Z-Level

### PR DESCRIPTION
**Pull request by Robustin**
How the fuck did #34734 get merged as a meme title PR with no changelog. 

I spent several hours adjusting off-station balance in my PR, having an off-station base is necessary during highpop (long post below and on forums about it). Cult is at its best when its not MAGICAL REVOLUTION with a conversion rush but playing it slow and low with a cult base is not possible on-station during highpop. You have 20+ assistants online, you have maints that can be fully explored within a couple minutes, then you get the inevitable "CULT IN MEDBAY MAINT" followed by a flurry of flashbang and gunspam down 1x1 hallways that leads to the cult having a <20% winrate during highpop. 

I added new counterplay for space basing in my cult PR, yet its been almost completely untested, so I would really appreciate it if Kor would let me see how my changes go before deciding to remove them after years of existence right as I'm tuning the entire cult mode. 

It's incredibly frustrating putting a lot of time and thought into these changes (that have overall been very popular) and most people reading this PR don't even realize the changes I have already implemented and might think I'm just going to let space basing play out like it has in the past - all because of a ridiculous meme PR that got no discussion and was implemented in the midst of existing changes to the same code after years of being untouched.

### CODE REVIEW

**Comment by Wilchenx** [comment]
goof's not wrong tho

**Comment by praisenarsie** [comment]
@Robustin fix his pr then by adding in the changelog directly

**Comment by Iamgoofball** [comment]
No matter how you slice it, fun enjoyable gameplay is not made by cultists building in the corner of the map. This was a direct request from Kor, he was going to PR it himself but I figured I'd do it for him as my popularity with the hip and kool players isn't important and it needed to happen.

**Comment by Iamgoofball** [comment]
apologies on the changelog though I made a fix PR for that

**Comment by optimumtact** [comment]
ironically now your meme title also doesn't explain your pr

**Comment by ExcessiveUseOfCobblestone** [comment]
What exactly about space bases is engaging?

**Comment by Robustin** [comment]
Space bases allowed the cult to do during highpop what it does during med/low pop. To organize in one location, centralize their resource, to coordinate and plan and maintain the most crucial aspect of the cult early game: Stealth. 

My cult update even added an extremely bright light and unique animation for teleport runes that had been used to travel from space, so when cultists return to the station there's about 60 seconds where the room with the teleport would be glowing so purple it would be impossible to miss (even through walls).

I believe this was an appropriate first step because this isn't Clock Cult where you have a camera pointed at your destination and you can teleport with 100% confidence. The teleport network has you going in blind and counterplay for space basing is as simple as standing outside the rune with a stun. I've played a shitload of cult and can tell you even the best teams rarely have more than 2 viable teleport destinations when traveling back to station, you'll have 2 decent teleport spots that meet the rare requirements of being both well-hidden yet able to let you back into an accessible part of the station (compared to the typical 3-4 "niche" teleport spots that leave you stranded on a solar control room because it was made in an access-restricted area by a cultist with that access).

So for a first step I felt that this was a good way to add risk to space-basing - making it much easier to find the cult's teleport runes and shut them down, the fewer runes, the more the cult becomes dependent on the remainder and thats when sec can basically camp the last couple runes and simply wipe the cult out the remainder of the cult. 

I had several other ideas like just removing stuff like manifest or summon cultist from space-areas, all this PR does is ensure that I sign up for mining during highpop cult because you simply cannot have a stealth team-antag on a station with 20 assistants. 

TL;DR you don't have to blow up a space base to shut down a cult that's using one. Teleport runes become vital highways and its not hard to turn those into ambushes. I added a significant nerf to space-basing in my PR that almost nobody has tested yet and I have many other nerfs planned if it continues to be a problem, but off-station bases are crucial for high-pop unless we want cult to basically be decided in the first 5 minutes of a round.

**Comment by ChangelingRain** [comment]
yeah idk why that got merged while robustin was testing a solution
